views/dashboard-application.hbs

```diff
@@ -47,58 +49,44 @@
 
         {{else}}
 
-
             <div class='ideahacks-form application-form container-fluid'>
+                <div class='row'>
+                    <div class='mainHeader col-md-12 text-center'>
+                        <h1>IDEAHACKS: HOME APPLICATIONS</h1>
+                    </div>
+                </div>
                 <!-- PERSONAL INFORMATION -->
                 <div class='row'>
                     <div class='header'>
-                        <h3>Personal </h3>
-                        <!-- <img id="img-side" src="/img/registration/logo.png" /> -->
+                        <h3>PERSONAL</h3>
                     </div>
                     <fieldset>
                         <div class='form-group col-md-5'>
-                            <label>First Name*</label>
+                            <label>FIRST NAME*</label>
                             <input class='form-control' type='text' name='firstName' placeholder='First Name' value='{{user.firstName}}'>
                         </div>
                         <div class='form-group col-md-5'>
-                            <label>Last Name*</label>
```

Comment:
Ideally, it would be nice to have last name immediately after first name, but it doesn't matter too much, I suppose.